### PR TITLE
Allow for the 'Translation' model to over extended/overridden

### PR DESCRIPTION
**Pull request by specialtactics**
Currently the 'Translation' model can't be extended/overriden using the Voyager::useModel() provider function (unlike other models). This PR resolves that issue.

### CODE REVIEW

**Comment by specialtactics** [comment]
Not really sure why StyleCI is failing, please feel free to let me know or just edit in place.

**Comment by emptynick** [comment]
Dont worry about StyleCI, it will fix itself once merged.

**Comment by specialtactics** [comment]
What's the sitrep on this one? Does this repo have any maintainers that are going to review the PR or what is the next step?

**Comment by emptynick** [comment]
We are all doing this in our free-time and we need the community to help us approve stuff.
So yes, the next step is that someone approves your changes.

**Review by marktopper — APPROVED** [review]

**Comment by specialtactics** [comment]
Thanks !!

**Pull request merged**
